hash/vsmile_cart.xml

```diff
@@ -837,9 +837,9 @@ V.Smile Smartbook Smartidges (need a Smartbook touch tablet connected to a regul
 			<feature name="cart_type" value="lilac" />
 			<feature name="u1" value="" /> <!-- EPOXY BLOB ROM --> <!-- 2 globs, uses /CS2 for 2nd glob -->
 			<feature name="u2" value="" /> <!-- EPOXY BLOB ROM -->
-			<dataarea name="rom" size="0x0800000">
-				<rom name="092003.u1" size="0x0400000" crc="402681b7" sha1="5ff1ba9a016d10e09408b5638aaf759cac2cc97a" offset="0x0000000" /> <!-- U1 is the same as U1 on 80-092000(US) -->
-				<rom name="092003.u2" size="0x0200000" crc="60cfb1b4" sha1="f37ad4dd774c26abb998f616ab7ad0adce83359d" offset="0x0400000" />
+			<dataarea name="rom" size="0x600000">
+				<rom name="092003.u1" size="0x400000" crc="402681b7" sha1="5ff1ba9a016d10e09408b5638aaf759cac2cc97a" offset="0x000000" /> <!-- U1 is the same as U1 on 80-092000(US) -->
+				<rom name="092003.u2" size="0x200000" crc="60cfb1b4" sha1="f37ad4dd774c26abb998f616ab7ad0adce83359d" offset="0x400000" />
```

Comment:
Once again, have you checked the slot code?  And please make put the offsets next to the sizes.